sabnzbd/filesystem.py

```diff
@@ -41,6 +41,7 @@
 from sabnzbd.constants import FUTURE_Q_FOLDER, JOB_ADMIN, GIGI
 from sabnzbd.encoding import correct_unknown_encoding
 from sabnzbd.utils import rarfile
+from sabnzbd.constants import DEF_FILE_MAX
```

Comment:
Can you add it to the import 2 lines above?

Reply:
Done